lib/ws_servers/api/handlers/on_settings_update.js

```diff
@@ -29,7 +29,7 @@ module.exports = async (server, ws, msg) => {
       fullScreen,
       packetWDDelay,
       autoResumeAOs,
-      showOptInModal,
+      // showOptInModal,
```

Comment:
Optional: We can just remove it if not required. Same, for other respective comments.

Reply:
Done